en/docs/includes/obtain-api-key.md

```diff
@@ -1,4 +1,4 @@
-After the APIs are exposed via WSO2 API Microgateway, you can invoke an API with a valid token(JWT or opaque access token) or an API key.  Let's use WSO2 API Microgateway's API key endpoint to obtain an API key in order to access the API.
+After the APIs are exposed via Choreo Connect, you can invoke an API with a valid token (JWT or opaque access token) or an API key.  Let's use Choreo Connect's API key endpoint to obtain an API key in order to access the API.
```

Comment:
Shall we remove this include file if it is not used in any other file

Reply:
Removed.